autopts/wid/has.py

```diff
@@ -0,0 +1,153 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2023, Oticon.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+import logging
+import re
+import sys
+
+from enum import IntFlag
+from autopts.pybtp import btp
+from autopts.ptsprojects.stack import get_stack
+from autopts.pybtp.types import WIDParams
+
+class PresetProperty(IntFlag):
+    BT_HAS_PROP_NONE      = 0
```

Comment:
this should probably be in autopts/pybtp/defs.py  (and I'd prefer it to be 0x00, 0x01, 0x02 etc)

Reply:
Done, but kept it locally in autopts/wid/has.py, since it is only used here.  Prefer no to "pollute" autopts/pybtp/defs.py.